src/themes/blueberry/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+import defaultTheme from '../../layout'
+import ExampleTypeConstructor from '../../exampleTypes/'
+
+import Input from './Input'
+import Checkbox from './Checkbox'
+import Fonts from './Fonts'
+import Style from './Style'
+import Table from './Table'
+import Button from './Button'
+import ButtonRadio from './ButtonRadio'
+import ListGroupItem from './ListGroupItem'
+import Wrap from './Wrap'
+import TagComponent from './TagComponent'
+import PagerItem from './PagerItem'
+import Adder from './Adder'
+
+export let Pager = props => (
+  <ExampleTypes.ResultPager Item={PagerItem} {...props} />
+)
+
+let baseTheme = {
+  ...defaultTheme,
+  Adder,
+  Wrap,
+  Fonts,
+  Style,
+  Table,
+  Button,
+  Input,
+  Checkbox,
+  RadioList: ButtonRadio,
+  'TagsInput.TagComponent': TagComponent,
```

Comment:
Why does this need to be scoped? What else uses TagComponent?

Reply:
you're right - nothing does, so it doesn't 👍